runtime/compiler/context.vim

```diff
@@ -3,7 +3,7 @@ vim9script
 # Language:           ConTeXt typesetting engine
 # Maintainer:         Nicola Vitacolonna <nvitacolonna@gmail.com>
 # Former Maintainers: Nikolai Weibull <now@bitwi.se>
-# Latest Revision:    2023 Dec 26
+# Last Change:        2024 Mar 29 by Enno Nagel
```

Comment:
I'd suggest this instead:

```
Contributors: Enno Nagel
Last Change: 2024 Mar 29
```

So that the contributor's name won't be lost at the next update.

Reply:
Well, Peter Collingbourne would already be victim to this pull request

Reply:
.. or, less drama, rather the written memory of his contribution in the compiler file

Reply:
I remember Bram wanting former maintainers to remain in the files. I think it's fair to leave all the names that are already there.

Reply:
Yes, seems so sensible I applied your suggestions in the latest iteration